lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -19,6 +19,10 @@
 %>
 
 <%
+  disable_unenroll_cert_states = [
+    'generating',
+    'ready',
```

Comment:
Should this list include `downloadable`? Or, more generally, are there more states that we want to include?